pkg/term/windows/windows_test.go

```diff
@@ -0,0 +1,3 @@
+// Dummy file
```

Comment:
My understanding is that only `windows.go` is needed for `go test ./...` to work.